### ISSUE TITLE
PURPOSE

### ISSUE THREAD

**Issue by lmarinucci** [misael77/IDEHAdataset#16]
La parola mancante e giusta è sempre FUNZIONE (es. funziona cultuale / funzione funeraria). Forse a volte FUNZIONE DI. Mi ricordavo di aver visto dei dati al maschile, quindi servirebbe SCOPO, ma nel sample non ci sono aggettivi al maschile. Se non riesci con un altro modello, cambio direttamente il template con "Il bene culturale ha finalità".

**Comment by LuanaBulla** [comment]
Bert sembra aver risolto  il problema (adesso quasi tutte le frasi hanno 'funzione' al loro interno). In alcuni casi però la parola è stata sostituita con 'valore' (ad esempio, "il bene culturale ha valore culturale" oppure "Il bene culturale ha valore civile"). Può andar bene lo stesso?

**Comment by lmarinucci** [comment]
No, la parola dovrebbe essere sempre FUNZIONE. L'ho quindi aggiunta come parola fissa nella risposta. I casi (eccezioni) in cui l'aggettivo nei dati sia al maschile (richiedendo quindi prima SCOPO) oppure sia un sostantivo (richiedendo DI dopo funzione) forse li possiamo correggere con il check grammaticale.
